app/components/show/controls_component.html.erb

```diff
@@ -43,6 +43,9 @@
                                      data: { turbo_method: 'post' } %></li>
         <% end %>
         <li><%= link_to 'Download CSV', item_descriptive_path(doc, format: :csv), class: 'dropdown-item' %></li>
+        <li><%= link_to 'Upload CSV', new_item_descriptive_path(doc),
+                                      class: 'dropdown-item',
```

Comment:
based on #3364 (which I just happen to pick up), do you want to just rename both options while you are here in this PR: "Download Cocina spreadsheet" and "Upload Cocina spreadsheet"

Reply:
#3364 should be marked as blocked and should not be worked on yet.